client/components/embed-container/index.jsx

```diff
@@ -112,6 +124,24 @@ function embedTumblr( domNode ) {
 	}, 30 );
 }
 
+function embedSlideshow( domNode ) {
+	debug( 'processing slideshow for', domNode );
+
+	// set global variable required by JetpackSlideshow
+	window.jetpackSlideshowSettings = {
+		spinner: 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/img/slideshow-loader.gif'
+	};
+
+	loadCSS( 'https://s0.wp.com/wp-content/mu-plugins/shortcodes/css/slideshow-shortcode.css' );
```

Comment:
probably want to avoid re-embedding this over and over for each slideshow during the session